### ISSUE TITLE
Minions send duplicate events to master when using salt-call

### ISSUE THREAD

**Issue by clallen** [saltstack/salt#53506]
### Description of Issue
Sending events from minions results in the master receiving duplicate events.
They are separate events (based on timestamp) from the same PID.
I can also tell the events are duplicates because I have reactors that will run twice when the events are received.

### Steps to Reproduce Issue
Run state.event on the master to watch for events
```
salt-run state.event testevent
```
Run event.send (or event.fire_master) on a minion
```
salt-call event.send testevent
```

Example output from my master:
```
testevent       {"_stamp": "2019-06-17T17:05:48.105232", "cmd": "_minion_event", "data": {"__pub_fun": "event.send", "__pub_jid": "20190617120548085634", "__pub_pid": 21253, "__pub_tgt": "salt-call"}, "id": "minion01", "tag": "testevent"}
testevent       {"_stamp": "2019-06-17T17:05:48.159265", "cmd": "_minion_event", "data": {"__pub_fun": "event.send", "__pub_jid": "20190617120548085634", "__pub_pid": 21253, "__pub_tgt": "salt-call"}, "id": "minion01", "tag": "testevent"}
```

This only seems to happen when events are sent using *salt-call*.

Both the master and minions are running the same build of Salt.
If there is any more info I can provide, please let me know.
### Versions Report
```
Salt Version:
           Salt: 2018.3.3

Dependency Versions:
           cffi: 1.6.0
       cherrypy: Not Installed
       dateutil: 2.8.0
      docker-py: Not Installed
          gitdb: 2.0.5
      gitpython: 2.1.11
          ioflo: Not Installed
         Jinja2: 2.10
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: 0.21.1
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.6.1
   mysql-python: Not Installed
      pycparser: 2.14
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.5 (default, Apr  9 2019, 16:02:27)
   python-gnupg: Not Installed
         PyYAML: 5.1
          PyZMQ: 17.0.0
           RAET: Not Installed
          smmap: 2.0.5
        timelib: Not Installed
        Tornado: 4.5.3
            ZMQ: 4.1.6

System Versions:
           dist: oracle 7.6
         locale: UTF-8
        machine: x86_64
        release: 4.14.35-1844.4.5.2.el7uek.x86_64
         system: Linux
        version: Oracle Linux Server 7.6
```

**Comment by xeacott** [comment]
This is both minion and master running on the same machine?

**Comment by clallen** [comment]
No, separate master and minion systems.  I've tried it from about a dozen different minions and I get consistent results with the 2018.3 ones.
We have a handful of 2016.11 systems, and it doesn't appear to happen with those.

**Comment by xeacott** [comment]
Gotcha, thanks for the information!

**Comment by clallen** [comment]
I have just discovered another aspect to the issue.  It only seems to happen when the minion is configured to use multiple (in our case 2) masters.
When /etc/salt/minion is configured like this, I get duplicate events:
```
master:
  - master1
  - master2
```
When configured like this, I get a single event:
```
master:
  - master1
```
Note that in both cases the event(s) are always going to master1.

**Comment by clallen** [comment]
FYI, This looks to be fixed in 2019.2.2.
